### ISSUE TITLE
Telegram - Persistent Profiles Issue

### ISSUE THREAD

**Issue by GadgetGeekNI** [kasmtech/workspaces-issues#395]
### Existing Resources

- [Y] Please search the existing issues for related problems
- [Y] Consult the product documentation : [Docs](https://kasmweb.com/docs/latest/index.html)
- [Y] Consult the FAQ : [FAQ](https://kasmweb.com/docs/latest/FAQ.html)
- [Y] Consult the Troubleshooting Guide : [Guide](https://kasmweb.com/docs/latest/guide/troubleshooting.html)
- [Y] Reviewed existing training videos: [Youtube](https://www.youtube.com/@KasmWorkspaces )

**Describe the bug**
End user reporting that she is constantly having to log back into Telegram on the container, the persistent profile doesn't seem to be working. Referenced in a previous ticket but no solution posted: https://github.com/kasmtech/workspaces-issues/issues/288 

**To Reproduce**
1. Install Telegram container, Set up persistent profile.
2. Open Container with persistent profile & Log in.
3. Delete Session
4. Open Container and see login not retained.

**Expected behavior**
1. Install Telegram container, Set up persistent profile.
2. Open Container with persistent profile & Log in.
3. Delete Session
4. Open Container and see login will be retained.

**Screenshots**
N/A

**Workspaces Version**
Version 1.13

**Workspaces Installation Method**
Multi-Server, AWS

**Client Browser (please complete the following information):**
 - OS: MacOS
 - Browser Chrome
 - Version  113.0.5672.126

**Workspace Server Information (please provide the output of the following commands):**
 - `uname -a` 
 Linux ip-10-1-101-156 5.10.0-15-cloud-amd64 # 1 SMP Debian 5.10.120-1 (2022-06-09) x86_64 GNU/Linux

 - `cat /etc/os-release`
 PRETTY_NAME="Debian GNU/Linux 11 (bullseye)"
NAME="Debian GNU/Linux"
VERSION_ID="11"
VERSION="11 (bullseye)"
VERSION_CODENAME=bullseye
ID=debian
HOME_URL="https://www.debian.org/"
SUPPORT_URL="https://www.debian.org/support"
BUG_REPORT_URL="https://bugs.debian.org/"
 
- `sudo docker info`
Client: Docker Engine - Community
 Version:    24.0.1
 Context:    default
 Debug Mode: false
 Plugins:
  buildx: Docker Buildx (Docker Inc.)
    Version:  v0.10.4
    Path:     /usr/libexec/docker/cli-plugins/docker-buildx
  compose: Docker Compose (Docker Inc.)
    Version:  v2.5.0
    Path:     /usr/local/lib/docker/cli-plugins/docker-compose

Server:
 Containers: 4
  Running: 4
  Paused: 0
  Stopped: 0
 Images: 4
 Server Version: 24.0.1
 Storage Driver: overlay2
  Backing Filesystem: extfs
  Supports d_type: true
  Using metacopy: false
  Native Overlay Diff: true
  userxattr: false
 Logging Driver: json-file
 Cgroup Driver: systemd
 Cgroup Version: 2
 Plugins:
  Volume: local
  Network: bridge host ipvlan macvlan null overlay
  Log: awslogs fluentd gcplogs gelf journald json-file local logentries splunk syslog
 Swarm: inactive
 Runtimes: io.containerd.runc.v2 runc
 Default Runtime: runc
 Init Binary: docker-init
 containerd version: 3dce8eb055cbb6872793272b4f20ed16117344f8
 runc version: v1.1.7-0-g860f061
 init version: de40ad0
 Security Options:
  apparmor
  seccomp
   Profile: builtin
  cgroupns
 Kernel Version: 5.10.0-15-cloud-amd64
 Operating System: Debian GNU/Linux 11 (bullseye)
 OSType: linux
 Architecture: x86_64
 CPUs: 2
 Total Memory: 1.902GiB
 Name: ip-10-1-101-156
 ID: 2e2ebe48-9aca-4688-ba28-fb24204337ac
 Docker Root Dir: /var/lib/docker
 Debug Mode: false
 Experimental: false
 Insecure Registries:
  127.0.0.0/8

 - `sudo docker ps | grep kasm`
 03d82bc39d9f   kasmweb/nginx:latest     "/docker-entrypoint.…"   About an hour ago   Up About an hour                   80/tcp, 0.0.0.0:443->443/tcp, :::443->443/tcp   kasm_proxy
bb063cd94eb6   kasmweb/manager:1.13.0   "/bin/sh -c '/usr/bi…"   About an hour ago   Up About an hour (healthy)         8181/tcp                                        kasm_manager
d4f45f12fb09   kasmweb/share:1.13.0     "/bin/sh -c '/usr/bi…"   About an hour ago   Up 41 seconds (health: starting)   8182/tcp                                        kasm_share
53f01ed84e59   kasmweb/api:1.13.0       "/bin/sh -c '/usr/bi…"   About an hour ago   Up About an hour (healthy)         8080/tcp                                        kasm_api

**Comment by j-travis** [comment]
I am able to reproduce this issue if using the `kasmweb/telegram:1.13.0` image. 

I suspect what is happening is the auto update process of telegram results in a profile that is un-usable by the older version of the program when you create a new session.  It probably is something like this.

1. User launches  `kasmweb/telegram:1.13.0` , the version in the image is at version  `x`
2. The users logs into telegram
3. The telegram auto updater kicks in and updates the app and the users profile to version `y`.
4. The user closes the session
5. The users opens a new session of `kasmweb/telegram:1.13.0`
6. The image still has version `x` installed, but cant read the profile for version `y` since its newer. 
7. Telegram treats the profile as corrupt and forces the user to log in again.


The likely best way around this is to use the rolling images which are built and published nightly . e.g  `kasmweb/telegram:1.13.0-rolling`. This should always have the latest version and would hopefully minimize the likelyhood of this occurring.


Another way which seemed to work in the short testing we did is to basically remove (or overwrite) the `/opt/Telegram/Updater` bin which seems to be in charge of the update process.  We used the [File Mapping](https://kasmweb.com/docs/latest/guide/workspaces.html#file-mapping) feature to quickly test and seemed to avoid the issue with the `kasmweb/telegram:1.13.0` image.

![image](https://github.com/kasmtech/workspaces-issues/assets/48762190/563af5d6-da59-4622-bdbc-5dd2b477d6ae)



Its worth a shot until a way is found to properly disable the updater.
